modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -425,21 +427,30 @@ sda.enkf.multisite <- function(settings,
         writeLines(runs.tmp[runs.tmp != ''], file.path(rundir, 'runs.txt'))
         paste(file.path(rundir, 'runs.txt'))  ## testing
         Sys.sleep(0.01)                       ## testing
-        PEcAn.workflow::start_model_runs(settings, write=settings$database$bety$write)
```

Comment:
This should probably be an `if` to choose between versions, not a replacement.

Reply:
Fixed.